src/Resources/config/profiler.xml

```diff
@@ -5,12 +5,15 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="wisembly.amqp.collector" class="Wisembly\AmqpBundle\DataCollector\AMQPDataCollector">
-            <tag name="data_collector" template="WisemblyAmqpBundle:Collector:MessageCollector" id="amqp_collector" />
-        </service>
+        <defaults public="false" />
```

Comment:
note ; not sure if this is correct, as a while ago all service subscribers couldn't be private. Don't know if it is still the case...

Reply:
it's fiiiiiine : https://github.com/symfony/symfony/pull/20953/files#diff-8604f72b4ae78c6c364c1af8c1dee17cL62